.github/comment-commands/retest.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+!/usr/bin/env bash
```

Comment:
Is removal of `#` intentional?

Reply:
```
2020-02-13T09:37:16.1836554Z /home/runner/work/hadoop-ozone/hadoop-ozone/.github/comment-commands/retest.sh: line 1: !/usr/bin/env: No such file or directory
```

Reply:
Not intentional. Just disappeared in the fog. Thanks to catch it.